docs/INSTALLATION.md

```diff
@@ -1,15 +1,52 @@
 # Installation
 
-<!-- TODO: write document
+Gdscript is a domain specific language and can thereby requires the godot game engine to be installed.
 
-  This document should describe what the student needs to install
-  to allow working on the track on their local system using the CLI.
+## General
 
-  You can include the installation instructions in this document, but
-  usually it is better to link to a resource with the official installation
-  instructions, to prevent the instructions from becoming outdated.
+Working on the track will require the godot engine being setup to run from the terminal.
 
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/installation`.
+You can install godot through [gitub][godot-github].
+Pick the version marked with your operating system and the version marked as non mono.
 
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+You can also install godot through [steam][godot-steam], but it is a bit harder to setup it to work as a cli tool, thereby it is not recommended while working on the track.
+
+## Windows
+
+### Github
+
+First you have to download the bit version which your operating system has (likely 64bit) of the engine from [github][godot-github].
```

Comment:
I appreciate the detailed instructions here, but would it be possible to link the Godot website instead? Again, this is my first Exercism track, so I don't know what is the policy here ;) Do we prefer adding all the necessary information?

Reply:
We can link to the website as a download location but removing the whole instruction I don't know.

Well as I see we have 2 options:

Either make each exercise into a godot project and be able to be opened and run within godot. Then would it be fine to link to godot and more or less say: Download and unzip and run.

Option 2 is that we make godot into a terminal tool, unfortunately for windows so is not that easy and Godot has no guide on how to do it. We could tell the user as you said earlier to use the docker container but I don't think that is the intended "exercism" way.

Reply:
But I can tell you as of now that this track is already having a bit of an edge case in how it works. In my ideal world so would godot have a system that when you download godot so does it automatically become an executable cli tool.

Reply:
The engine installs are available on the official website so I think we can link there so students can get the Godot engine for their OS and then mention the OS specifics for setting up environment variables for *nix, Windows, and MacOS (no more than a sentence or two per OS). We can then break the Steam installation off as an alternative install option at the bottom.